src/utils/tests/expressions.spec.js

```diff
@@ -29,10 +29,6 @@ describe("expressions", () => {
     it("sanitizes 2 quotes", () => {
       expect(sanitizeInput('"3"')).toEqual('"3"');
     });
-
-    it("sanitizes forward slashes", () => {
-      expect(sanitizeInput("foo\\\\")).toEqual("foo\\\\\\\\");
-    });
   });
```

Comment:
lets add the two cases

Reply:
yep, we need to add the two test cases!

  1.  Add N\u{61}N as watch expression
  2.  Add '\u{61}' as watch expression

Expected results:

  1.  NaN
  2.  "a"

Reply:
Test cases added